generic/BRGeneric.c

```diff
@@ -360,7 +366,7 @@ genWalletCreateTransfer (BRGenericWallet wallet,
                               ? GENERIC_TRANSFER_SENT
                               : GENERIC_TRANSFER_RECEIVED));
 
-    genWalletAddTransfer (wallet, transfer);
+    // We don't add `transfer` to `wallet`.  Let an explicit `genWalletAddTransfer` do that.
```

Comment:
I think I get it - we don't add it when we create the transfer - it get's created once we pull it down from blockset?  Or when we get a response from the submit?

Reply:
Yes.